src/deroff.rs

```diff
@@ -538,12 +538,311 @@ impl Deroffer {
         unimplemented!()
     }
 
+    fn quoted_arg(&self) -> bool {
+        unimplemented!()
+    }
+
+    fn request_or_macro(&mut self) -> bool {
+        // self.skip_char();
+        self.s = self.skip_char(&self.s, None).to_owned();
+
+        let s0 = &self.s[1..2];
+
+        match s0 {
+            "\\" => {
+                // if self.str_at(1) == Some('"') {
```

Comment:
Is this still needed?

Reply:
#75 isn't merged yet, once it is i can move this to the new interface

Reply:
Can you put a TODO here? At least we know we need can come back to check later.

Reply:
#75 is merged, I believe this patch needs a rebase.

Reply:
These have been fixed :)